api/envoy/config/rbac/v3/rbac.proto

```diff
@@ -274,12 +300,16 @@ message Permission {
     // URI template path matching.
     // [#extension-category: envoy.path.match]
     core.v3.TypedExtensionConfig uri_template = 13;
+
+    // Matches against metadata from either dynamic state or route configuration. Preferred over the
+    // ``metadata`` field as it provides more flexibility in metadata source selection.
+    SourcedMetadata sourced_metadata = 14;
```

Comment:
Please mark and docmuent that the previous `metadata` is deprecated. In most cases, we will not keep two field with same featue. Thanks.

Reply:
Done, Thanks!